torch/CMakeLists.txt

```diff
@@ -259,9 +259,6 @@ if(USE_DISTRIBUTED)
     if(USE_NCCL)
       list(APPEND TORCH_PYTHON_LINK_LIBRARIES __caffe2_nccl)
     endif()
-    if(USE_UCC)
-      list(APPEND TORCH_PYTHON_LINK_LIBRARIES __caffe2_ucc)
-    endif()
```

Comment:
Don't we have the same problem as the above comment with NCCL?

Reply:
I am not very sure, but my understanding is, because of this:
https://github.com/pytorch/pytorch/blob/706b420a52e8e5773c46cbf70a761fe365771549/torch/CMakeLists.txt#L273-L277
and
https://github.com/pytorch/pytorch/blob/706b420a52e8e5773c46cbf70a761fe365771549/torch/csrc/cuda/Module.cpp#L857-L869
PyTorch exposes some NCCL APIs directly (not through a PG), and the that's the reason why
```
list(APPEND TORCH_PYTHON_LINK_LIBRARIES __caffe2_nccl)
```
is needed. But UCC does not have such thing.